resources/packs/core/mondoo_eol.go

```diff
@@ -29,7 +29,11 @@ func (p *mqlMondooEol) GetDate() (*time.Time, error) {
 
 	// get new advisory report
 	// start scanner client
-	scannerClient, err := newAdvisoryScannerHttpClient(mcc.ApiEndpoint, mcc.Plugins, ranger.DefaultHttpClient())
+	rangerClient, err := rangerclient.NewRangerClient()
```

Comment:
this should not be created here but when the mcc object is created. Then it is just `mcc.HttpClient`

Reply:
gotcha...i added the httpclient so that it can be set up way up at the scan/shell/etc subcommands